### ISSUE TITLE
JavaFx main dispatcher fails to initialize on JDK8

### ISSUE THREAD

**Issue by elizarov** [Kotlin/kotlinx.coroutines#816]
See https://youtrack.jetbrains.com/issue/KT-28088

This code:

```
import kotlinx.coroutines.*
import kotlinx.coroutines.javafx.*

object UI : CoroutineScope {
    override val coroutineContext = Dispatchers.JavaFx
}

fun main(args: Array<String>) {
    UI.launch {
        println("Hi")
    }
}
```

with kotlinx.coroutines version 1.0.1 produces:

```
Exception in thread "main" java.lang.ExceptionInInitializerError
	at kotlinx.coroutines.javafx.JavaFxDispatcherKt.getJavaFx(JavaFxDispatcher.kt:23)
	at UI.<clinit>(Scratch.kt:5)
	at ScratchKt.main(Scratch.kt:9)
Caused by: java.lang.NoSuchMethodException: javafx.application.Platform.startup(java.lang.Runnable)
	at java.lang.Class.getMethod(Class.java:1786)
	at kotlinx.coroutines.javafx.JavaFxDispatcherKt.initPlatform(JavaFxDispatcher.kt:137)
	at kotlinx.coroutines.javafx.JavaFx.<clinit>(JavaFxDispatcher.kt:81)
	... 3 more
```

**Comment by elizarov** [comment]
@eXponenta It is not fixed yet. It is a bug in the #798 commit.

**Comment by Romanow88** [comment]
@elizarov is there an update when this will be fixed? Right now this blocks us from migrating to Koltin 1.3.

**Comment by qwwdfsad** [comment]
@Romanow88 it will be fixed in 1.0.1 in a week or two.

**Comment by Romanow88** [comment]
@qwwdfsad you mean 1.0.2?

**Comment by qwwdfsad** [comment]
Yes, thanks.
Meanwhile, you can update on 1.0.0 because this bug was introduced in 1.0.1

**Comment by jcornaz** [comment]
@qwwdfsad Will a `1.0.2` be released? Or do we have to wait for `1.1.0`?

**Comment by elizarov** [comment]
@jcornaz There will be no `1.0.2`.
